lib/capybara/playwright/driver.rb

```diff
@@ -10,7 +10,10 @@ def initialize(app, **options)
         @browser_runner = BrowserRunner.new(options)
         @page_options = PageOptions.new(options)
         if options[:timeout].is_a?(Numeric)
-          @default_navigation_timeout = options[:timeout] * 1000
+          timeout = options[:timeout] * 1000
+
+          @default_timeout = timeout
```

Comment:
`options[:timeout]` is generally used as read timeout, let's change the name of option like `@default_timeout = option[:playwright_default_timeout]` or let's consider another solution like this

```
page.with_playwright_page(default_timeout: 2000) do |page|
  page.get_by_label(...) # only waits 2 seconds
end
```